eastlake/steps/newish_metacal/metacal/metacal_fitter.py

```diff
@@ -159,7 +138,73 @@ def _do_all_metacal(self, mbobs_list, data=None):
         output = eu.numpy_util.combine_arrlist(datalist)
         return output
 
-    def _do_one_metacal(self, mbobs):
+    def _do_one_metacal_ngmixv2(self, mbobs):
```

Comment:
@esheldon can you check this method over? It works locally on my machine just fine, but I want to make sure I have  things set reasonably.

Reply:
Our priors are usually not constraining on T and flux, so PriorGuesser might  not work well

Reply:
I recall in fitvd I fell back on some factor of the PST T value when moments failed.

Reply:
Good idea!